<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Anvi" failures="6" tests="29" name="pageTestEvents.RegisterPageEvents" time="80.527" errors="0" timestamp="2024-11-16T18:16:04 CST" skipped="12">
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testPageTitle" time="0.026"/>
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistration" time="1.974"/>
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistration" time="1.909"/>
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistration" time="3.153">
    <skipped/>
  </testcase> <!-- testUserRegistration -->
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistrationjson" time="3.143">
    <skipped/>
  </testcase> <!-- testUserRegistrationjson -->
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistrationjson" time="3.117">
    <failure message="expected [user name cannot be only numeric] but found [password_mismatch:The two password fields didn’t match.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [user name cannot be only numeric] but found [password_mismatch:The two password fields didn’t match.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at pageTestEvents.RegisterPageEvents.registerUser(RegisterPageEvents.java:42)
at pageTestEvents.RegisterPageEvents.testUserRegistrationjson(RegisterPageEvents.java:76)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- testUserRegistrationjson -->
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistration" time="2.025"/>
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistrationjson" time="4.206"/>
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistrationjson" time="1.844"/>
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistrationjson" time="3.026">
    <skipped/>
  </testcase> <!-- testUserRegistrationjson -->
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistrationjson" time="3.125">
    <failure message="expected [password length is less than 8] but found [password_mismatch:The two password fields didn’t match.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [password length is less than 8] but found [password_mismatch:The two password fields didn’t match.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at pageTestEvents.RegisterPageEvents.registerUser(RegisterPageEvents.java:42)
at pageTestEvents.RegisterPageEvents.testUserRegistrationjson(RegisterPageEvents.java:76)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- testUserRegistrationjson -->
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistrationjson" time="3.061">
    <failure message="expected [password length doesnot meet] but found [password_mismatch:The two password fields didn’t match.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [password length doesnot meet] but found [password_mismatch:The two password fields didn’t match.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at pageTestEvents.RegisterPageEvents.registerUser(RegisterPageEvents.java:42)
at pageTestEvents.RegisterPageEvents.testUserRegistrationjson(RegisterPageEvents.java:76)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- testUserRegistrationjson -->
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistrationjson" time="3.093">
    <skipped/>
  </testcase> <!-- testUserRegistrationjson -->
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistrationjson" time="3.082">
    <skipped/>
  </testcase> <!-- testUserRegistrationjson -->
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistrationjson" time="3.135">
    <skipped/>
  </testcase> <!-- testUserRegistrationjson -->
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistrationjson" time="1.907"/>
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistrationjson" time="3.145">
    <skipped/>
  </testcase> <!-- testUserRegistrationjson -->
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistrationjson" time="1.987"/>
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistration" time="3.112">
    <failure message="expected [user already exists] but found [password_mismatch:The two password fields didn’t match.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [user already exists] but found [password_mismatch:The two password fields didn’t match.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at pageTestEvents.RegisterPageEvents.registerUser(RegisterPageEvents.java:42)
at pageTestEvents.RegisterPageEvents.testUserRegistration(RegisterPageEvents.java:68)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.retryFailed(TestInvoker.java:269)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:76)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- testUserRegistration -->
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistrationjson" time="1.996"/>
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistration" time="3.466">
    <skipped/>
  </testcase> <!-- testUserRegistration -->
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistration" time="4.188"/>
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistration" time="3.079">
    <failure message="expected [password length doesnot meet] but found [password_mismatch:The two password fields didn’t match.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [password length doesnot meet] but found [password_mismatch:The two password fields didn’t match.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at pageTestEvents.RegisterPageEvents.registerUser(RegisterPageEvents.java:42)
at pageTestEvents.RegisterPageEvents.testUserRegistration(RegisterPageEvents.java:68)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- testUserRegistration -->
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistrationjson" time="3.127">
    <failure message="expected [password cannot be only numeric] but found [password_mismatch:The two password fields didn’t match.]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [password cannot be only numeric] but found [password_mismatch:The two password fields didn’t match.]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at pageTestEvents.RegisterPageEvents.registerUser(RegisterPageEvents.java:42)
at pageTestEvents.RegisterPageEvents.testUserRegistrationjson(RegisterPageEvents.java:76)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- testUserRegistrationjson -->
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistration" time="2.040"/>
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistrationjson" time="3.129">
    <skipped/>
  </testcase> <!-- testUserRegistrationjson -->
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistrationjson" time="3.119">
    <skipped/>
  </testcase> <!-- testUserRegistrationjson -->
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistrationjson" time="3.132">
    <skipped/>
  </testcase> <!-- testUserRegistrationjson -->
  <testcase classname="pageTestEvents.RegisterPageEvents" name="testUserRegistration" time="3.181">
    <skipped/>
  </testcase> <!-- testUserRegistration -->
</testsuite> <!-- pageTestEvents.RegisterPageEvents -->
